common/transport/mqtt/src/mqtt_base.ts

```diff
@@ -61,11 +103,12 @@ export class MqttBase extends EventEmitter {
         },
         connecting: {
           _onEnter: (connectCallback) => {
-            this._connectClient((err, connack) => {
+            this._connectClient((err, conAck) => {
```

Comment:
>conAck [](start = 38, length = 6)

Since the MQTT Spec has this as one word, CONNACK, I'm not inclined toward this change.

Reply:
I think you're right.  I should put it back.

---
In reply to: [431343870](https://github.com/Azure/azure-iot-sdk-node/pull/808#discussion_r431343870) [](ancestors = 431343870)